catboost/libs/model/coreml_helpers.cpp

```diff
@@ -2,20 +2,44 @@
 #include "model_build_helper.h"
```

Comment:
Please add categorical features support to CoreML tutorial. Use titanic dataset for it (catboost.datasets.titanic())

Reply:
it is in catboost/tutorials/apply_model/core_ml